modules/workload-identity/main.tf

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 Google LLC
```

Comment:
Can you switch the headers to use the same comment style as other files in the repo?

Reply:
Done